spec/custom_rules/AmazonMQBrokerEncryptOptionsRule_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
-require 'password_rule_spec_helper'
+require 'custom_rule_spec_helper'
```

Comment:
we could probably jam all of this stuff into spec_helper going forward? (to avoid 45 deltas :) )

Reply:
True, I considered that, however all the functions in `custom_rule_spec_helper` are more geared toward the types of rules that @phelewski created in his password rule framework.  I just found the [boolean_base_rule.rb](lib/cfn-nag/custom_rules/boolean_base_rule.rb) however, and that may fulfill my needs for the "missing property base rule".  I'll do more research and possibly remove most of this PR.  😄